Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM quay.io/giantswarm/alpine:3.9-giantswarm
+FROM quay.io/giantswarm/alpine:3.10-giantswarm
 
 USER root
 
-RUN apk add --no-cache ca-certificates
+# bind-tools is required by the init container to use dig.
+RUN apk add --no-cache ca-certificates bind-tools
```

Comment:
Using dig instead of nslookup because it supports setting multiple name servers. Rather than creating an `alpine-dig` image I'm just adding it to the current image.